### PR TITLE
Setup Local Development with localtunnel and Slack

### PR DESCRIPTION
**Pull request by angelocordon**
Setup framework for being able to develop locally and connect to a Slack channel via their APIs.

### CODE REVIEW

**Review by stain88 — CHANGES_REQUESTED** [review]
So, i think Step 4 of "Set up greetbot" needs to be moved up to before Step 2 of "Grab your tokens for your .env file", since you don't get the tokens until the app is installed, but you can't install the app without a bot user. There's also a typo in `Slash Commnands`.
Still need to go through the events, as this branch is now out of date with development, and at the very least, `package.json` is different, so haven't run the install process yet.

File: package.json [hunk 1]
```diff
@@ -22,6 +24,6 @@
     "app-module-path": "^2.x"
   },
   "devDependencies": {
-    "nodemon": "^1.17.1"
+    "localtunnel": "^1.8.3",
```
Comment (stain88) [inline package.json]: Trailing comma here also seems to be an issue for me

Comment (stain88) [reply package.json]: actually, why has nodemon gone? needs to stay in

**Comment by stain88** [comment]
Another thing to note: due to our use of `await/async`, people must have node version ^8.x installed - don't know if that should be in Chapter 1 setup, or at the top of the greetbot page.

**Comment by stain88** [comment]
One more: when setting up commands, need to make sure "Escape channels, users, and links sent to your app" is checked, otherwise our parser breaks

File: package.json (hunk 1 above)
Comment (angelocordon) [reply package.json]: Great catch, I think that's a merge conflict on my end, will put it back in 👍

**Review by stain88 — APPROVED** [review]

**Pull request merged**